GameData/RP-1/Contracts/Earth Observation 1/EOSIRRad1.cfg

```diff
@@ -117,3 +118,258 @@ CONTRACT_TYPE
 		}
 	}
 }
+
+CONTRACT_TYPE
+{
+	name = EOSIRRad2	
+	title = Infrared Radiometry Satellite
+	group = EOS
+	agent = Federation Aeronautique Internationale
+	
+	description = <b>Program: Early Earth Observation Satellites<br>Type: <color=blue>Optional</color></b><br><br>After the success of your first weather satellite, the US Weather Bureau wants additional satellites available for more timely weather observations with better coverage of Earth.  Launch additional weather satellites to fulfill the Weather Bureau's request.<br><br><br><br><b><color="green">Make sure your craft contains an Infrared Radiometry 1 experiment and an Early TV camera.  Science requirements can be met over multiple launches.</color></b><br><br>Real life examples: TIROS-2 through TIROS-8.
+
+	synopsis = Launch the first weather satellite with the Infrared Radiometry experiment and an Early TV camera.
+
+	completedMessage = Success! The images provided from this satellite have proven that weather satellites can be extremely effective in helping meteorologists predict the weather.
+
+	sortKey = 402
+
+	cancellable = true
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 1
+	maxSimultaneous = 1
+	deadline = 0
+
+	targetBody = HomeWorld()
+
+	// ************ REWARDS ************
+	prestige = Trivial       // 1.0x
+	advanceFunds = 0
+	rewardScience = 0
+	rewardFunds = 0
+	failureFunds = 0
+	rewardReputation = 30
+	failureReputation = 0 // was @rewardReputation
+
+	// ************ REQUIREMENTS ************
+
+	REQUIREMENT
+	{
+		name = ProgramActive
+		type = ProgramActive
+		program = EarlyEOS
+	}
+
+	REQUIREMENT	
+	{	name = CompleteContract
+		type = CompleteContract
+		contractType = EOSIRRad1
+		title = Completed first weather satellite
+	}
+
+	PARAMETER
+	{
+		name = ScienceSatIRRad
+		type = VesselParameterGroup
+		title = Additional weather satellite
+		define = ScienceSatIRRad
+
+		PARAMETER
+		{
+			name = NewVessel
+			type = NewVessel
+			title = Launch a new vessel
+			hideChildren = true
+		}
+		PARAMETER 
+		{
+			name = Crewmembers
+			type = HasCrew
+			minCrew = 0
+			maxCrew = 0
+			title = Uncrewed
+			hideChildren = true
+		}
+
+		PARAMETER
+		{
+			name = OrbitSequence
+			type = Sequence
+			title = Survive in orbit for 2 days and transmit science
+
+			PARAMETER
+			{
+				name = Orbit
+				type = Orbit
+				maxApA = 1000000
+				minPeA = 500000
+				maxEccentricity = 0.005
+				maxInclination = 75
+				disableOnStateChange = true
+				title = Achieve orbit between 500km and 1,000km in altitude with no more than .005 eccentricity and inclination lower than 75 degrees.
+			}
+
+			PARAMETER
+			{
+				name = Duration
+				type = Duration
+				duration = 2d

```

Comment:
See previous review re: durations better long on EOS contracts and short on CA contracts.